Modules/Quest/QuestieQuest.lua

```diff
@@ -379,7 +379,12 @@ function QuestieQuest:AcceptQuest(questId)
                 Questie:SendMessage("QC_ID_BROADCAST_QUEST_UPDATE", questId)
             end,
             function() QuestieQuest:PopulateObjectiveNotes(quest) end,
-            function() QuestieTracker:Update() end,
+            function() QuestieTracker:Update()
+                -- This is necessary to call it again to update the trackers formatting
```

Comment:
If this is true for all the other double calls below, can't the formatting be split away from `QuestieTracker:Update()`?